_posts/2018-06-22-pulp_qe_intern_update.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 2018 Pulp QE Intern Update #2
+author: Ragavendran Balakrishnan
+tags:
+  - pulpqe
+---
+
+## Journey so Far
+
+The first two weeks of my Red Hat journey was a roller coaster ride with lots of take in's, which I
```

Comment:
A "take-in" is a deception or con. I don't think "take-in" is the right word here. Maybe "takeaways"? Or lessons, learning experiences, etc?